src/views/ApartmentApp/components/StudentApplication/components/HallSelection/components/HallListItem/index.js

```diff
@@ -26,7 +26,7 @@ import ClearIcon from '@material-ui/icons/Clear';
  * @param {Number} props.index The index of this list item
  * @param {Number} props.hallRank The rank assigned to this hall by the user
  * @param {String} props.hallName The name of the apartment hall
- * @param {ApartmentChoice[]} props.preferredHalls Array of apartment choices
+ * @param {ApartmentChoice[]} props.apartmentChoices Array of apartment choices
```

Comment:
Actually, I think HALLChoices would be more accurate. Of course, this is what we call it in the API, isn't it...

Reply:
actually there are a few reasons why we use apartmentChoice instead of hallChoice, but of course I forgot since it was a while ago. It has something to do with being specific to the apartment part and differentiating between future housing-related projects. 
ALso because halls include all sorts of things that are not even dorms, so it is best to keep it specific.

Reply:
ApartmentHalls have ApartmentChoices, 

DormHalls might have DormChoices some day 😆

Reply:
Okay, yeah, makes sense!